library/alloc/src/vec/mod.rs

```diff
@@ -49,7 +49,26 @@
 //! v[1] = v[1] + 5;
 //! ```
 //!
+//! # Memory layout
+//!
+//! For non-zero-sized values, a [`Vec`] will use the [`Global`] allocator for its allocation. It is
```

Comment:
```suggestion
//! For non-zero-sized types, a [`Vec`] uses the [`Global`] allocator for its allocation. It is
```
present tense is clearer